content/help/faq/security/k8s-health-checks.md

```diff
@@ -2,23 +2,16 @@
 title: How can I use Kubernetes liveness and readiness for pod health checks when mutual TLS is enabled?
 weight: 50
 ---
-If mutual TLS is enabled, http and tcp health checks from the kubelet will
-not work since the kubelet does not have Istio-issued certificates.
 
-As of the Istio 1.0 release, we support the [`PERMISSIVE` mode](/docs/tasks/security/mtls-migration)
-for Istio services so they can accept both http and mutual TLS traffic
-when this mode is turned on. This can solve the health checking issue.
-Please keep in mind that mutual TLS is not enforced since others can
-communicate with the service with http traffic.
+If mutual TLS is enabled, http and tcp health checks from the kubelet will not work since the kubelet does not have Istio-issued certificates.
 
-You can use a separate port for health checks and enable mutual TLS only
-on the regular service port. Refer to [Health Checking of Istio Services](/help/ops/setup/app-health-check/)
-for more information.
+As of the Istio 1.1 release, we have several options to solve this issue. 
 
-Due to the risk of a new feature, we do not turn on above feature by default. The future rollout
-plan is tracked on [GitHub issue](https://github.com/istio/istio/issues/10357).
+You can use probe rewrite to rewrites the liveness/readiness probe such that the probe request will redirected to application directly. And you can also use a separate port for health checks and enable mutual TLS only on the regular service port. Please refer to [Health Checking of Istio Services](/help/ops/setup/app-health-check/#mutual-tls-is-enabled) for more information.
 
-To mitigate the risk, another workaround is to use a [liveness command](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-command)
+Moreover, we support the [`PERMISSIVE` mode](/docs/tasks/security/mtls-migration) for Istio services so they can accept both http and mutual TLS traffic when this mode is turned on. This can solve the health checking issue. Please keep in mind that mutual TLS is not enforced since others can communicate with the service with http traffic.
+
+Another workaround is to use a [liveness command](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#define-a-liveness-command)
```

Comment:
Can we move this cmd option as second one? Since this is better for for people caring security?

Reply:
This one has it's own limitation too, i.e., customer may/could not be able to install curl to their services. And we documented the limitation of the permissive option as well, so i would prefer we keep the order.

Reply:
IMO, the limitation of commands available are less important than the security implications. After some conversations and some cases study, I feel, people coming to this page, should be those who really want to get both mTLS protection and liveness prober working at the same time. `PERMISSIVE` is more of migration and not an option targeted for these audiences above.

Reply:
It's not all about importance, it's also related to feasibility/overhead. In some container, you simply can't install curl. And for permissive, we already mentioned the limitation. No big deal tho, feel free to update later if you feel it's necessary.